proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -270,6 +270,9 @@ message WorkflowExecutionInfo {
     // (-- api-linter: core::0203::immutable=disabled
     //     aip.dev/not-precedent: field_behavior annotation is not yet used in this repo --)
     temporal.api.workflow.v1.WorkflowExecutionVersioningInfo versioning_info = 98;
+    // This filed is to record the transition history when the transition history is cleaned up due to disabling transition history
+    repeated VersionedTransition previous_transition_history = 99;
+    VersionedTransition previous_transition_history_break_point = 100;
```

Comment:
Is it too late to create a separate message for all those transition history related fields? I think `transition_history` should always be empty now?

Reply:
hmm I guess I am also ok with not having the separate message. Those two fields won't be needed in the longer so we can just deprecate them. Let's add some comments for that then.